srcpkgs/oomox/template

```diff
@@ -15,28 +17,33 @@ maintainer="Tim Wanders <timwanders241@gmail.com>"
- https://github.com/themix-project/oomox-gtk-theme/archive/$_oomox_gtk_theme_ver.tar.gz>oomox-gtk-theme-$_oomox_gtk_theme_ver.tar.gz"
+	https://github.com/themix-project/oomox/archive/$version.tar.gz>oomox-$version.tar.gz
+	https://github.com/themix-project/oomox-archdroid-icon-theme/archive/$_droid_icon_ver.tar.gz>oomox-archdroid-icon-theme-$_droid_icon_ver.tar.gz
+	https://github.com/themix-project/oomox-gnome-colors-icon-theme/archive/$_gnome_icon_ver.tar.gz>oomox-gnome-colors-$_gnome_icon_ver.tar.gz
+	https://github.com/themix-project/oomoxify/archive/$_oomoxify_ver.tar.gz>oomoxify-$_oomoxify_ver.tar.gz
+	https://github.com/nana-4/materia-theme/archive/v$_materia_theme_ver.tar.gz>materia-theme-$_materia_theme_ver.tar.gz
+	https://github.com/themix-project/oomox-gtk-theme/archive/$_oomox_gtk_theme_ver.tar.gz>oomox-gtk-theme-$_oomox_gtk_theme_ver.tar.gz
+	https://github.com/actionless/arc-theme/archive/$_arc_theme_ver.tar.gz>arc-theme-$_arc_theme_ver.tar.gz
+	https://github.com/base16-builder/base16-builder/archive/$_base16_builder_ver.tar.gz>base16-builder-$_base16_builder_ver.tar.gz"
 checksum="
- a32e9159cc772f953ba63caf478271b2152c0daf54db82ae52658d5e18c6e719
- d6e91b9f6ea48c0831eaf8cbf266ad2bac672e02d5d28a91caafcf181956f6fa
- b7bf90cd61f10b8f7afc1442ecde4ba11bdafcb1e48471c81c568d3f6a04f82a
- ebbe38cc62889c58d7b3eb4e21f6c5e2ae54fc16761cf65e99a05571067536ae
- dee2a0b38f8fbca3d2980af89f40309f0cb646fad25b88eab1fae5582047160e
- 98bb87ba18581cd29e1469e6a441f0113d6700e9540492bc4663d0e2d877018b"
-
+	b6df65d89c41b6cf5992b922f260b850f59fcf1a6d17b27a08fbf36f5ef0fab0
+	eb4a5c9edcaca47c22fcb3356268a40fa131c98c489d7fc8f03aff81b1533576
+	01d09aaf4079843051c7329d1f01114f71eea11a08a142c27c88026ae1fb6a79
+	093929410afd0ebc9108778f60c22f91202e1c85eb098fcfd08cc2d4dab5e490
+	4d9bec034767e2a08f843e1dd61bdf7ead97621c2977886a9288fb898342578f
+	e54e97e617f05babd3ae9b674073898879af37df4cc623f4d42b94c3053da9fa
+	cdb315f1b85c19c1334752da22f02096be58a8086f7000d7ff92461710a9e022
+	d18596e11f196931fec183e587d52a388a60f6fe5011741dd7f77a3216e3fe2b"
 post_extract() {
 	rm -rf archdroid-icon-theme gnome-colors-icon-theme oomoxify \
 		gtk-theme materia-theme
-	cp -r ../archdroid-icon-theme-$_droid_icon_ver archdroid-icon-theme
-	cp -r ../gnome-colors-icon-theme-$_gnome_icon_ver gnome-colors-icon-theme
-	cp -r ../oomoxify-$_oomoxify_ver oomoxify
-	cp -r ../materia-theme-$_materia_theme_ver materia-theme
-	cp -r ../oomox-gtk-theme-$_oomox_gtk_theme_ver gtk-theme
+	mv ../archdroid-icon-theme-$_droid_icon_ver archdroid-icon-theme
+	mv ../gnome-colors-icon-theme-$_gnome_icon_ver gnome-colors-icon-theme
+	mv ../oomoxify-$_oomoxify_ver oomoxify
+	mv ../materia-theme-$_materia_theme_ver materia-theme
+	mv ../oomox-gtk-theme-$_oomox_gtk_theme_ver gtk-theme
+	mv ../arc-theme-$_arc_theme_ver arc-theme
+	mv ../base16-builder-$_base16_builder_ver base16-builder
 }
 
 do_install() {
```

Comment:
and you can refactor it to use make: https://github.com/themix-project/oomox/blob/master/packaging/arch/PKGBUILD#L94